### ISSUE TITLE
ORIGINS does not work to change the allowed hosts

### ISSUE THREAD

**Issue by djcastaldo** [joplin-vieweb/joplin-vieweb#34]
I can't find a way to contact you, but when ORIGINS var is changed in the docker compose file, it not chnaging the allowed hosts.

**Comment by djcastaldo** [comment]
ALLOWED_HOSTS and CSRF_TRUSTED_ORIGINS from the django is ignoring the ORIGINS var from the docker compose file.  It is always set to 'localhost' and a local ip '192.168.128.63'.   I need it to allow a different url.

**Comment by djcastaldo** [comment]
nevermind, i had to delete the volumes before it would re-read the config.

**Comment by gri38** [comment]
I open this bug: you're right, we would expect the ORIGINS var is changed, it's processed.
Your workaround is good: delete volume or settings file.

**Comment by gri38** [comment]
Fix in 2.4.3.

Upate:
```bash
sudo docker-compose down
sudo docker-compose pull
sudo docker-compose up
```
